server/server.go

```diff
@@ -140,6 +140,42 @@ func (server *HTTPServer) readReportForCluster(writer http.ResponseWriter, reque
 	}
 }
 
+func (server *HTTPServer) deleteOrganizations(writer http.ResponseWriter, request *http.Request) {
+	orgIds, err := readOrganizationIDs(writer, request)
+	if err != nil {
+		// everything has been handled already
+		return
```

Comment:
If read organization returns an error, maybe we need to send a 404 response or something like that ??

Reply:
More like `400 Bad Request` because if `readOrganizationIDs` fails, it means that the parameters sent along with the request were invalid. It tells us nothing about reports associated with the organizations.

Reply:
`400` is better there

Reply:
Agree. TYVM